ports/geckolib/binding_tools/regen.py

```diff
@@ -315,16 +341,61 @@ def build(objdir, target_name, debug, debugger, kind_name=None,
         for ty in current_target["void_types"]:
             flags.append("--raw-line")
             flags.append("pub enum {} {{}}".format(ty))
-    if "servo_arc_types" in current_target:
-        for ty in current_target["servo_arc_types"]:
+    if "servo_nullable_arc_types" in current_target:
+        for ty in current_target["servo_nullable_arc_types"]:
             flags.append("--blacklist-type")
             flags.append("{}Strong".format(ty))
             flags.append("--raw-line")
-            flags.append("pub type {0}Strong = ::sugar::refptr::Strong<{0}>;".format(ty))
+            flags.append("pub type {0}Strong = ::sugar::ownership::Strong<{0}>;".format(ty))
+            flags.append("--blacklist-type")
+            flags.append("{}BorrowedOrNull".format(ty))
+            flags.append("--raw-line")
+            flags.append("pub type {0}BorrowedOrNull<'a> = ::sugar::ownership::Borrowed<'a, {0}>;".format(ty))
```

Comment:
We could use `include!` instead, and have regen.py generate that file.

I plan to make bingen read from a json file instead of arguments at some point.